validation/genesis/monorepo-outputs.sh

```diff
@@ -0,0 +1,25 @@
+set -e
+
+echo "Inferring and selecting correct Node version"
+export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || printf %s "${XDG_CONFIG_HOME}/nvm")"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
+nvm use
+
+echo "Running install command"
+eval $1
```

Comment:
It seems risky to blindly execute a shell command that a user is passing in and has complete control over formatting. It might be better for the user to provide an enum-type value, which our code uses to select from a list of known, predefined commands to execute